lib/account/account-manager.js

```diff
@@ -36,6 +75,25 @@
  */
 
 /**
+ * Lifetime for how long a file should last. This can
+ * contain as many properties as needed to adjust the
+ * time.
+ *
+ * @typedef {Object} accountManager~lifeTime
+ * @property {string} Hours, Minutes, Seconds
+ *
+ */
+
+/**
+ * Information the Account Holder sends to complete the
+ * login process.
+ *
+ * @typedef {Object} accountManager~loginCompleteInfo
+ * @property {string} password
```

Comment:
I changed it to secureHash as this matches what we are saving in the login file to compare when the client sends in their securely hashed password.